portal-impl/src/content/Language.properties

```diff
@@ -2466,7 +2466,8 @@ layout.types.embedded=Embedded
 layout.types.embedded.description=Show content from another website.
 layout.types.panel=Panel
 layout.types.panel.description=Create a page with predefined applications and navigation.
-layout.types.portlet=Layout
+layout.types.portlet=Widget Page
+layout.types.portlet.description=Build a page from a layout and adding widgets and content.
```

Comment:
@brianchiu i think this one needs to be corrected as well - 

as i read it, two ways of understanding the phrase show up:

1. Build a page from a layout, adding widgets and content
1. Build a page from a layout and add widgets and content

The verb tenses don't match in the way that it's currently used.